mm2src/adex_cli/src/scenarios/init_mm2_cfg.rs

```diff
@@ -0,0 +1,331 @@
+                }
+            })
+            .with_placeholder(default_password)
+            .with_help_message(
+                "Type \"none\" to leave it blank and use limited service\n\
+                 Your passphrase; this is the source of each of your coins' private keys. KEEP IT SAFE!",
+            )
+            .prompt()
+            .map_err(|error| {
+                error!("Failed to get passphrase: {error}");
+            })
+            .map(|value| if "none" == value { None } else { Some(value) })?;
+
+        Ok(())
+    }
+
+    #[inline]
+    fn inquire_rpc_password(&mut self) -> Result<(), ()> {
+        let allow_weak_password = self.allow_weak_password;
+        let validator = move |password: &str| {
+            if let Some(false) = allow_weak_password {
+                match password_policy::password_policy(password) {
+                    Err(error) => Ok(Validation::Invalid(error.into())),
+                    Ok(_) => Ok(Validation::Valid),
+                }
+            } else {
+                Ok(Validation::Valid)
+            }
+        };
+        let default_password = Self::generate_password()?;
+
+        self.rpc_password = Text::new("What is the rpc_password:")
+            .with_help_message("Your password for protected RPC methods (userpass)")
+            .with_validator(validator)
+            .with_default(default_password.as_str())
+            .with_placeholder(default_password.as_str())
+            .prompt()
+            .map_err(|error| {
+                error!("Failed to get rpc_password: {error}");
+            })?
+            .into();
+        Ok(())
+    }
+
+    fn generate_password() -> Result<String, ()> {
+        let pg = PasswordGenerator {
+            length: 8,
+            numbers: true,
+            lowercase_letters: true,
+            uppercase_letters: true,
+            symbols: true,
+            spaces: false,
+            exclude_similar_characters: true,
+            strict: true,
+        };
+        let mut password = String::new();
+        while password_policy::password_policy(&password).is_err() {
+            password = pg
+                .generate_one()
+                .map_err(|error| error!("Failed to generate password: {error}"))?;
+        }
+        Ok(password)
+    }
+
+    #[inline]
+    fn inquire_allow_weak_password(&mut self) -> Result<(), ()> {
+        self.allow_weak_password = Confirm::new("Allow weak password:")
+                .with_default(false)
+                .with_placeholder("No")
+                .with_help_message(r#"If true, will allow low entropy rpc_password. If false rpc_password must not have 3 of the same characters in a row, must be between 8-32 characters in length, must contain at least one of each of the following: numeric, uppercase, lowercase, special character (e.g. !#$*). It also can not contain the word "password", or the chars <, >, and &. Defaults to false."#)
```

Comment:
`If false rpc_password must not have 3 of the same characters in a row, must be between 8-32 characters in length`

password must be at least 8 characters, but could have more than 32.
https://github.com/KomodoPlatform/atomicDEX-API/pull/1329/files#diff-cfd6c8a052dbc90f76f53398162840d84a70325a185acd369a7cf4ab0edbfa2aR117

Could you remove 32 characters restriction?

Reply:
Sure, great, thank you, I just copied it from documentation as it was )

Reply:
Done